src/main/java/graphql/validation/rules/UniqueDirectiveNamesPerLocation.java

```diff
@@ -57,17 +58,23 @@ public void checkOperationDefinition(OperationDefinition operationDefinition) {
     }
 
     private void checkDirectivesUniqueness(Node<?> directivesContainer, List<Directive> directives) {
-        Set<String> names = new LinkedHashSet<>();
-        directives.forEach(directive -> {
-            String name = directive.getName();
-            if (names.contains(name)) {
+        Set<String> directiveNames = new LinkedHashSet<>();
+
+        for (Directive directive : directives) {
+            String directiveName = directive.getName();
+            GraphQLDirective graphQLDirective = getValidationContext().getSchema().getDirective(directiveName);
+
+            if (graphQLDirective == null) continue;
```

Comment:
please use fully formed `if` statements

```
if (graphQLDirective == null) {
    continue;
}
````

more lines :  yes - more readable :  also yes

Reply:
by using the GraphQL java code formatter it will be automatically handled.